docs/UserGuide.md

```diff
@@ -64,6 +64,7 @@ FriendlyLink Fields are described below.
 #### NRIC
 NRIC is a unique identifier given to all Singaporeans.
 * NRIC is case-insensitive
+* All input NRIC should conform to the ICA-specified format, i.e. one single letter followed by seven digits followed by one letter.
```

Comment:
For this it might be better to be more specific. Follow the `MESSAGE_CONSTRAINTS` string under the `Nric` class. Namely,

> The structure of the NRIC should be @xxxxxxx#, where:
> 1. @ is a letter that can be"S", "T", "F", "G" or "M"
> 2. xxxxxxx is a 7-digit serial number
> 3. \# is a letter from A-Z